data/json/recipes/recipe_ammo.json

```diff
@@ -117,11 +117,11 @@
   "reversible": false,
   "autolearn": true,
   "book_learn": [[ "glassblowing_book", 5 ]],
+  "qualities":[
+    {"id":"GRIP","level":3}
```

Comment:
I don't know where the current elitist attitude towards amateur contributors crept in from, but you shouldn't ever have to worry about that **at all**.

Go ahead and fix it; I'll not stand by and let anyone give you trouble for doing a good job.